compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -29,17 +30,12 @@ impl<'cx, 'tcx> crate::MirBorrowckCtxt<'cx, 'tcx> {
         borrow_span: Span,
         borrow_desc: &str,
     ) -> DiagnosticBuilder<'tcx, ErrorGuaranteed> {
-        let mut err = struct_span_err!(
-            self,
+        self.infcx.tcx.sess.create_err(CannotUseWhenMutablyBorrowed {
             span,
-            E0503,
-            "cannot use {} because it was mutably borrowed",
```

Comment:
Why did you change the wording from here to "cannot use `{$desc}` when mutably borrowed"?

Reply:
I read some of the examples in the ftl file and there were certain placeholders there so since desc would be put in here in the message I thought {desc} would be also substituted in the ftl file from the struct passed in.

Reply:
I might be completely wrong, feel free to correct me

Reply:
I meant, why did you change the wording? The old phrasing was "because it was mutably borrowed", the new phrasing is "when mutably borrowed"? I think we should separate diagnostics migration from tweaks of wording.